<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetWork</class>
 <widget class="QWidget" name="NetWork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NetWork</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(238, 255, 250, 200);
border-width: 4px;
/*边框圆角半径*/
border-radius: 10px;</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>通过</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>失败</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; font-weight:600;&quot;&gt;测试步骤:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体';&quot;&gt;A. 在写号步骤开机过程中同步初始化网口PHY；&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体';&quot;&gt;B. 上位机通过usb指令&lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Calibri';&quot;&gt;Ping &lt;/span&gt;&lt;span style=&quot; font-family:'宋体';&quot;&gt;设备&lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Calibri';&quot;&gt;IP &lt;/span&gt;&lt;span style=&quot; font-family:'宋体';&quot;&gt;；&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; font-weight:600;&quot;&gt;判定：&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体';&quot;&gt;上位机ping成功自动判定为 &lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Calibri';&quot;&gt;Pass &lt;/span&gt;&lt;span style=&quot; font-family:'宋体';&quot;&gt;。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	image: url(:/ProductTest_360/Tip.jpg);
border-style: outset;
border-width: 4px;
border-radius: 10px;
border-color: rgba(255, 170, 0, 30);
}
QLabel:hover{
background-color:rgba(100,255,100, 100);
border-color: rgba(255, 170, 0, 255);
color:rgba(0, 0, 0, 255);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    /* 背景渐变色*/
   /* background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #E0E0E0, stop: 1 #FFFFFF);*/
background-color: rgba(238, 255, 250, 0);
    /* 边框 */
    border: 2px solid/* blue*/;
border-color: rgba(0, 170, 255, 150);

    /* 倒角 */
    border-radius: 8px;

    /* 就像墙上挂着的两个相框,margin指的是相框与相框的距离
       padding指的是每个相框里照片与相框边框的距离 */
    margin-top: 20px;
}</string>
   </property>
   <property name="title">
    <string>ping</string>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 4px;
/*边框圆角半径*/
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ping</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>381</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    /* 背景渐变色*/
   /* background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #E0E0E0, stop: 1 #FFFFFF);*/
background-color: rgba(238, 255, 250, 0);
    /* 边框 */
    border: 2px solid/* blue*/;
border-color: rgba(0, 170, 255, 150);

    /* 倒角 */
    border-radius: 8px;

    /* 就像墙上挂着的两个相框,margin指的是相框与相框的距离
       padding指的是每个相框里照片与相框边框的距离 */
    margin-top: 20px;
}</string>
   </property>
   <property name="title">
    <string>结果显示</string>
   </property>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>361</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ProductTest_360.qrc"/>
 </resources>
 <connections/>
</ui>
